library/Icingadb/ProvidedHook/Reporting/SlaReport.php

```diff
@@ -71,6 +71,12 @@ protected function fetchReportData(Timerange $timerange, array $config = null)
 
         if (isset($config['breakdown']) && $config['breakdown'] !== 'none') {
             switch ($config['breakdown']) {
+                case 'hour':
+                    $interval = new DateInterval('PT1H');
+                    $format = 'h:i:s A';
```

Comment:
Why not just use `H:i:s` as a format or is there some technical context to this that I don't get?